core/src/test/scala/io/snappydata/core/LocalTestData.scala

```diff
@@ -79,7 +79,7 @@ object LocalSparkConf {
         setAppName(getClass.getName)
     conf.set("snappy.store.optimization", "true")
     conf.set("spark.sql.inMemoryColumnarStorage.batchSize", "3")
-    conf.set("spark.snappy.analyticsService.interval" , "3")
+    conf.set("spark.snappy.analyticsService.interval" , "1000")
```

Comment:
Please avoid setting this for all the tests, and instead just override `newSparkConf` method of SnappyFunSuite in your specific tests?